packages/html-reporter/src/testErrorView.tsx

```diff
@@ -22,9 +22,10 @@ import { ImageDiffView } from '@web/shared/imageDiffView';
 
 export const TestErrorView: React.FC<{
   error: string;
-}> = ({ error }) => {
+  testId?: string;
+}> = ({ error, testId }) => {
   const html = React.useMemo(() => ansiErrorToHtml(error), [error]);
-  return <div className='test-error-view test-error-text' dangerouslySetInnerHTML={{ __html: html || '' }}></div>;
+  return <div className='test-error-view test-error-text' data-testId={testId} dangerouslySetInnerHTML={{ __html: html || '' }}></div>;
```

Comment:
```suggestion
  return <div className='test-error-view test-error-text' data-testid={testId} dangerouslySetInnerHTML={{ __html: html || '' }}></div>;
```

Thats the first time we do it this way - curious why it works in the first place with the wrong capitalisation.